src/components/question.cjsx

```diff
@@ -35,18 +35,27 @@ module.exports = React.createClass
     QuestionActions.removeAnswer(@props.id, answerId)
     @props.sync()
 
+  moveAnswer: (answerId, direction) ->
+    QuestionActions.moveAnswer(@props.id, answerId, direction)
+    @props.sync()
+
```

Comment:
Does the component need to call `.sync` here? It looks like the store calls it internally whenever `moveAnswer` is called?